scripts/report-parser.js

```diff
@@ -15,8 +15,7 @@ var directory = process.argv.slice(2)[0];
 
 const getJunitTestReports = (directory) => {
 	if (!fs.existsSync(directory)) {
-		console.log(`##vso[task.logissue type=warning]${directory} does not exist`);
-		return [];
+		throw new Error(`${directory} does not exist`);
```

Comment:
To make it clearer if the invalid entry is an empty string or some other value that's not obviously a path.
```suggestion
		throw new Error(`Directory '${directory}' does not exist`);
```